src/soc/intel/alderlake/acpi/tcss.asl

```diff
@@ -690,7 +690,7 @@ Scope (\_SB.PCI0)
 		Method (_STA, 0x0, NotSerialized)
 		{
 			If (DME0 == 1) {
-				Return (0x0F)
+				Return (0x0B)
```

Comment:
This is not a good solution. We just hide errors, not missing drivers. When booting Windows without a dock, al TBT related devices are working properly without yellow bangs.

Reply:
I had issues where it either could not find drivers for the devices, or it found them but they errored out with code 43, but I understand. Will revert.

Reply:
Reverted in 1dccfab207a274aab62cfe6f32666f7f30b39f46